release/rllib_tests/learning_tests/yaml_files/impala/impala-breakoutnoframeskip-v5.yaml

```diff
@@ -18,8 +18,5 @@ impala-breakoutnoframeskip-v5:
         num_workers: 10
         num_envs_per_worker: 5
         clip_rewards: True
-        lr_schedule: [
-            [0, 0.0005],
-            [20000000, 0.000000000001],
-        ]
+        lr: 0.0005
```

Comment:
I noticed that 20000000, twntymillion in words, is never reached so to not confuse this for 2M, we should just leaved it out.

Reply:
this sounds good.